src/js/fluxcontroller.js

```diff
@@ -708,13 +709,16 @@ define(function (require, exports, module) {
             }
 
             var throttledName = name + THROTTLED_ACTION_SUFFIX,
+                throttledWithDelayName = name + THROTTLED_WITH_DELAY_ACTION_SUFFIX,
```

Comment:
I don't think we should make a new global suite of throttled actions. Instead, in each module, we should just instantiate a new throttled function which calls the action.